app/views/pages/signup.ejs

```diff
@@ -0,0 +1,35 @@
+<%- include('../partials/header'); %> 
+<%- include('../partials/navbar'); %>
+
+<main class="container-xl">
+    <main class="container-xl">
+        <% if (err) { %>
+            <div class="alert alert-danger" role="alert"><%= err %></div>
+        <% } %>  
+        <h1>Sign Up</h1>
+        <form action="/signup" method="post">
+            <div class="mb-3">
+                <label for="username" class="form-label">Email</label>
+                <input id="email" name="email" type="email" class="form-control" autocomplete="email" required>
+            </div>
+            <div class="mb-3">
+                <label for="new-password" class="form-label">Password</label>
+                <input id="new-password" name="password" type="password" class="form-control" autocomplete="new-password" required name="password" 
+                pattern="^(?=.*\d)(?!.*\s).{8,100}$" 
```

Comment:
This pattern allows for a simple password like "12345678", is this what is intended?

Reply:
Added condition such that password now needs at least 1 letter. I also updated the backend validation for that too.